lms/static/js/spec/search/search_spec.js

```diff
@@ -611,19 +611,7 @@ define([
                 $('.search-button').trigger('click');
                 // cancel search
                 $('.cancel-button').trigger('click');
-                AjaxHelpers.respondWithJson(requests, {
-                    total: 1337,
-                    access_denied_count: 12,
-                    results: [{
-                        data: {
-                            location: ['section', 'subsection', 'unit'],
-                            url: '/some/url/to/content',
-                            content_type: 'text',
-                            excerpt: 'this is a short excerpt',
-                            course_name: ''
-                        }
-                    }]
-                });
+                AjaxHelpers.skipResetRequest(requests);
```

Comment:
Thanks @jzoldak. I've improved the comment above this line of code, and added a better docstring on the helper function.